.clang-format

```diff
@@ -0,0 +1,212 @@
+---
+Language:        Cpp
```

Comment:
It is hard to read. Lets keep here only the keys wit not default values.

Reply:
Yes we can.

Reply:
Done

Reply:
Thank you, but you forgot to add 

```
ConstructorInitializerIndentWidth: 2
ConstructorInitializerAllOnOneLineOrOnePerLine: true
```

Reply:
Yes, just saw :sweat: Done